RELEASING.md

```diff
@@ -24,3 +24,11 @@ make release
 Once a release is prepared 2 Pull Request will be generated. They have to be merged in order. First the one for the release and then the one for the next development version (it should be a draft PR). The automation will take care of the rest.
 
 > **_NOTE:_** Please review the changes in case there are errors if the PR were modified after the automation ran.
+
+#### Versioning standard
```

Comment:
For consistency capitalize Standard or just leave it as Versioning

Reply:
Removed.